keyboards/redox_w/redox_w.c

```diff
@@ -1,8 +1,8 @@
 #include "redox_w.h"
 
 void led_init(void) {
-	DDRD  |= (1<<1);
-	PORTD |= (1<<1);
+	DDRD  |= (1<<0) | (1<<1);
```

Comment:
ideally, these should be changed to GPIO commands. 

https://docs.qmk.fm/#/internals_gpio_control

Reply:
Ok, the modification has been completed